modules/serverless-install-serving-web-console.adoc

```diff
@@ -14,23 +14,22 @@ image::serving-installed-operator.png[Installed Operators page]
 . Click the *Create Knative Serving* button.
 +
 image::serverless-create-serving.png[Knative Serving tab]
-. In the *Create Knative Serving* page, you can choose to configure the `KnativeServing` object by using either the default form provided, or by editing the YAML.
-+
-* Using the form is recommended for simpler configurations that do not require full control of `KnativeServing` object creation.
-+
-Optional. If you are configuring the `KnativeServing` object using the form, make any changes that you want to implement for your Knative Serving deployment.
-+
-After you complete the form, click *Create*.
+. In the *Create Knative Serving* page, you can choose to configure the `KnativeServing` object by using either the form provided, or by editing the YAML.
```

Comment:
s/using either the form provided, or.../using the form provided, or..../
To me using `either` and `or` seems redundant. 
Do you need the comma after `provided`?

Reply:
Do you need to describe what the user can configure?

Reply:
That's why I have a note about advanced configuration options :slightly_smiling_face: this PR adds a new section to discuss the optional configurations you can do, they're just not discussed here because they're not mandatory, but it's mentioned that there are docs for those options.

We are also currently working through some issues re things showing up for configuration here that should actually be hidden (they're included in the CRD for upstream), so there will be follow up work to do for this, especially around OCP 4.5.